### PR TITLE
Update Xterm package

### PR DESCRIPTION
**Pull request by JL2210**

### CODE REVIEW

**Comment by JL2210** [comment]
@uberhacker Done.

File: packages/xterm.rb [hunk 1]
```diff
@@ -1,29 +1,45 @@
...
-     armv7l: '90ab39740d658445bca5c4368cce5f112b7572a667211a5498fa78f01ec72671',
-       i686: '97bfadabd2ac88155db24befc32a68f7f5213e25dd2481cf10dc0d1ce64ae67e',
-     x86_64: '27454e6a81b865fb7f75ad92a472b3823cb3f1f18c4fbb40a287bc3296384d61',
   })
 
+  depends_on 'pcre2'
+  depends_on 'libxpm'
+  depends_on 'libxaw'
+  depends_on 'freetype'
+  depends_on 'libxinerama'
```
[the remark this answers is not in the document]
Comment (uberhacker) [reply packages/xterm.rb]: Please don't self resolve conversations.

The sommelier package does include all but one of the dependencies in this package:
```
$ crew install sommelier
sommelier: Sommelier works by redirecting X11 and Wayland programs to the built-in ChromeOS wayland server.
https://chromium.googlesource.com/chromiumos/containers/sommelier
version 1382ce084cc4
The following packages also need to be installed: 
xorg_proto xcb_proto libxau pthread_stubs libxcb libxtrans libx11 docbook sgml_common docbook_xml libgpgerror python27 libxml2_python libgcrypt docbook_xsl libxslt libwayland xkeyboard_config libxext libxfixes libxv libbsd libice libsm libxrender libxt libpng bz2 krb5 libedit xzutils sqlite libtirpc python3 fuse libpstat fskit lvm2 libcap_ng vdev util_linux six glib freetype_sub harfbuzz gperf freetype fontconfig libfontenc libxmu libxpm libxi libxinerama libxdamage libxtst libpciaccess libxkbfile libxshmfence libxcursor libdmx libfs libxaw libxfont2 libxft libxrandr libxres libxvmc libxxf86dga libxxf86vm libxcomposite libxscrnsaver xorg_lib wayland_protocols eudev libdrm wayland elfutils mesa libepoxy libxfont pixman xinit libxkbcommon libunwind font_util glproto xcb_util xkbcomp xwayland psmisc 
Do you agree? [Y/n] 
```
Please remove libxpm, libxaw, freetype and libxinerama dependencies.

[the remark this answers is not in the document]
Comment (JL2210) [reply packages/xterm.rb]: Fair enough. You're really on top of this :).

[the remark this answers is not in the document]
Comment (JL2210) [reply packages/xterm.rb]: Done.

**Pull request merged**